terragrunt/env/production/app/terragrunt.hcl

```diff
@@ -9,7 +9,6 @@ dependencies {
 dependency "ecr" {
   config_path = "../ecr"
   mock_outputs_allowed_terraform_commands = ["init", "fmt", "validate", "plan", "show"]
-  mock_outputs_merge_strategy_with_state  = "shallow"
```

Comment:
Could you keep this attribute - it'll let us add new mock outputs in the future.